toolbox/issuer.go

```diff
@@ -60,13 +59,10 @@ func NewIssuerManager(config keycloak.Config, keyContextIssuerDomain interface{}
 	}, nil
 }
 
-func (im *issuerManager) GetIssuer(ctx context.Context) (OidcVerifierProvider, error) {
-	if rawValue := ctx.Value(im.keyContextIssuerDomain); rawValue != nil {
```

Comment:
As keyContextIssuerDomain is not used anymore, you should remove it from issuerManager.
(it is some kind of "delete cascade" starting by removing second parameter from api.New(...))